pkgs/development/python-modules/pyside/tools.nix

```diff
@@ -13,7 +13,8 @@ buildPythonPackage rec {
 
```

Comment:
Can you also change the source to https://pypi.org/project/PySide/?

This url wasn't stable anyway.

Reply:
Maybe drop the old comment above as well.

Reply:
The "PySide Tools" packet is not available via pypi/pip as it seems (and it does not have a ```setup.py``` either to support it). 

I am not sure why the archive should not be stable, as it is based on a tagged GitHub release. I replaced the fetch function however with the GitHub specific one.

I also dropped the old comment.